tests/acceptance/pageObjects/FilesPageElement/sharingDialog.js

```diff
@@ -311,7 +327,8 @@ module.exports = {
      * @returns {Promise.<string[]>} Array of autocomplete items
      */
     getShareAutocompleteItemsList: async function () {
-      const webElementIdList = await this.getShareAutocompleteWebElementIdList()
+      let webElementIdList = await this.getShareAutocompleteWebElementIdList()
```

Comment:
`getShareAutocompleteWebElementIdList()` returns empty array by default, if it doesnot find anything. So, this step is useless here.